src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -22,8 +22,9 @@
 
   <!-- General xunit options -->
   <PropertyGroup>
+    <XunitResultsFileName>testResults.xml</XunitResultsFileName>
     <XunitCommandLine>xunit.console.netcore.exe $(TargetFileName)</XunitCommandLine>
-    <XunitOptions>$(XunitOptions) -xml .\testResults.xml</XunitOptions>
+    <XunitOptions>$(XunitOptions) -xml $(XunitResultsFileName)</XunitOptions>
```

Comment:
Based on your other work you made me realize that we are going to have a conflict with the testResults.xml file as well when we start batching. We are going to need to figure out a work around for that.